exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/DateTypeFunctions.java

```diff
@@ -424,26 +424,35 @@ public void eval() {
     public static class AgeTimeStampFunction implements DrillSimpleFunc {
         @Param TimeStampHolder left;
         @Param TimeStampHolder right;
+        @Workspace org.joda.time.PeriodType periodType;
```

Comment:
Looks like the period type is constant. Can it be made a `static final` in, say, the `DateUtilities` class? Is the object thread-safe so that it can be shared instead of generating the code into each query (and duplicating the definition)? Looks like we need one for months/days/millis and another for months/days?

Reply:
To be honest, I was not able to come up with a good name for this constant :)